zone/zone_npc_factions.cpp

```diff
@@ -164,14 +173,30 @@ void Zone::LoadFactionAssociations(const std::vector<uint32>& npc_faction_ids)
 
 	for (const auto& e : npc_faction_ids) {
 		for (const auto& f : m_npc_factions) {
+			bool found=false;
 			if (e == f.id && f.primaryfaction > 0) {
-				faction_ids.emplace_back(f.primaryfaction);
+				for (auto a : m_faction_associations) {
+					if (a.id == f.primaryfaction) {
+						found=true;
+						LogError("Association [{}] already loaded", a.id);
+					break;
```

Comment:
Weird indentation.

Reply:
I don't seee this issue locally - tabs in github?

Reply:
Probably tabs versus spaces.

Reply:
Fixed in next push.